### PR TITLE
Added xbox autopkg file

### PR DESCRIPTION
**Pull request by sanchitgarg**

### CODE REVIEW

File: targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        version : <%- sdkVersion %>;
+        title: "PlayFab Xbox Platform C++ Sdk for Visual Studio <%- vsYear %>";
+        summary: "PlayFab is the unified backend platform for games and everything you need to build and operate your game, all in one place, so you can focus on creating and delivering a great player experience.";
+        authors: "PlayFab";
+        owners: "PlayFab";
+        licenseUrl: "https://raw.githubusercontent.com/PlayFab/XPlatCppSdk/master/LICENSE";
+        projectUrl: "https://github.com/PlayFab/XPlatCppSdk";
+        iconUrl: "https://playfab.com/assets/img/playfab-mark.png";
+        requireLicenseAcceptance: false;
+        description: "Authentication, in-game commerce, player data, title data, inventory, characters, statistics, leaderboards, analytics and reporting, friends, multiplayer, matchmaking, tournaments, cloud script, trading, real-time event handling, player management, live ops, and server hosting for all major platforms/devices and games of any scale. This sdk gives your game the ability log into PlayFab and access cloud data and services.";
```
Comment (Vicodex) [inline targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs]: >description [](start = 8, length = 11)

You may add "Xbox" tag here too for better association and indexing

File: targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        releaseNotes: "https://api.playfab.com/releaseNotes/#<%- sdkDate %>";
+        copyright: "Copyright 20<%- sdkYear %>";
+        language: "C++";
+        tags: { PlayFab, Baas, Paas, JSON, REST, HTTP, SSL, API, cloud, liveops, game, gamedev, native, xbox };
```
Comment (Vicodex) [inline targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs]: >xbox [](start = 104, length = 4)

Good job adding this tag here!

File: targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                build\Xbox\Durango\Profile\XPlatXbox.lib;
+            }
+            symbols: {
+                build\Xbox\Durango\Profile\XPlatXbox.pri;
+            }
+        };
+    };
+
+    targets {
+        Libraries += ws2_32.lib;
```
Comment (Vicodex) [inline targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs]: > ws2_32.lib [](start = 20, length = 11)

Do we still need this?

Comment (sanchitgarg) [reply targets/XPlatCppSdk/source/com.playfab.xplatxboxsdk.v141.autopkg.ejs]: Yes, QoS needs it.

**Pull request merged**